modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -271,9 +258,7 @@
       "NEXT_STEP": "Next",
       "BACK": "Previous",
       "CREATE": "Create",
-      "RETRACT": "Retract",
       "RETRACT_DELETE": "Retract/Delete",
```

Comment:
@krinnewitz The button is still named "Retract/Delete". Please just name it "Delete" (und use the translation key "DELETE") as at the end of the day, pressing the button will delete the events. While this might involve retraction, that is a detail not of concern to the user as it also results in deletion.